src/routes/+page.svelte

```diff
@@ -2,187 +2,62 @@
 	// NOTE: Might want to move assets outside of /src/ since that would build it differently
 	// NOTE: Since these are static images we have to import them like this, since the svelte compiler apparently does not
 	// allow for things like <img src="/bla/blorg/blirg.png">
-	import img from "$lib/images/strommingPog.png"
-	import vid from "$lib/videos/video.mp4"
-
-	import { Grid, gridHelp } from "$lib/js/svelte-grid";
+	
+	import Header from "./Header.svelte";
+	import InfoTab from "./tabs/InfoTab.svelte";
+	import VideoPlayerTab from "./tabs/VideoPlayerTab.svelte";
+	import { fade } from 'svelte/transition';
+	import Tabs from "./tabs/Tabs.svelte";
 
 	const id = () => "_" + Math.random().toString(36).substr(2, 9);
```

Comment:
same as above?

Reply:
idk if this should be in the videoplayer tab mby?

Reply:
leave for now